manifests/metallb/metallb.yaml

```diff
@@ -364,7 +364,7 @@ spec:
             secretKeyRef:
               name: memberlist
               key: secretkey
-        image: quay.io/metallb/speaker:main
+        image: quay.io/metallb/speaker:v0.12.1
```

Comment:
Is this a temporary change to work around some issue?

Reply:
metallib:main (apparently) doesn't accept '-config' anymore, breaking this deployment. We need to controller:main also needs to be changed to controller:v0.12.1 (at least until someone can sort out what happened to metallib)

Reply:
updated both images